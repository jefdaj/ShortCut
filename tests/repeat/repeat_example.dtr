result = "repeat example not written yet"
