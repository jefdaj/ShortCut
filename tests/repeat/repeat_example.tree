.
|-- cache
|-- exprs
|   `-- str
|       `-- 27770322da.str -> ../../cache/lines/...
`-- vars
    `-- result -> ../exprs/str/27770322da.str

4 directories, 2 files
