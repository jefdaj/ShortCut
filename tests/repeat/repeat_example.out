"repeat example not written yet"
